js/main.js

```diff
@@ -3,6 +3,30 @@
 
```

Comment:
**[OPTIONAL]**

- [ ] You have some disabled linters in your main.js file :fearful: Kindly remove them and fix your linters manually instead of disabling them :+1: Because linters prevent you making those errors which can break your whole application :thinking: